src/sentry/models/group.py

```diff
@@ -626,7 +612,59 @@ def is_resolved(self):
         return self.get_status() == GroupStatus.RESOLVED
 
     def has_replays(self):
-        return get_replay_count_for_group(self) > 0
+        def make_snuba_params_for_replay_count_query():
+            return SnubaParams(
+                organization=self.project.organization,
+                projects=[self.project],
+                user=None,
+                start=datetime.now() - timedelta(days=14),
+                end=datetime.now(),
+                environments=[],
+                teams=[],
+            )
+
+        def _cache_key(issue_id):
+            return f"has_replays:{issue_id}"
```

Comment:
I think we should namespace this key.  issue_id is an integer (right?).  there could be other products in the future that ask this same question.

`issues:has_replays:{issue_id}` or `group:has_replays:{issue_id}`

Reply:
👍🏼